src/components/ChatLog.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import ChatEntry from './ChatEntry';
+import './ChatLog.css';
+import PropTypes from 'prop-types';
+
+
+const ChatLog = ({entries,updateLikes}) => {
+
+    return (
+    <div className="chat-log">
+        {entries.map((entry) => (
+            <ChatEntry 
+            sender={entry.sender}
+            body = {entry.body}
+            timeStamp = {entry.timeStamp} 
+            updateLikes ={updateLikes}
+            />
+        ))}
+    </div>  
+    );         
+};
+
+ChatLog.propTypes = {
+    entries: PropTypes.array.isRequired,
```

Comment:
You could make this even more specific by using `arrayOf` and `shape` functionality in `Prop-types` to say that it needs to be an array of objects that have a certain shape.